requirements.txt

```diff
@@ -1 +1,3 @@
-ops >= 2.2.0
+ops>=2,<3
```

Comment:
Can you pin these?

Reply:
Can you pin to an exact version?

Reply:
I discussed with @jdkandersson about pinning the major version here https://github.com/canonical/jenkins-agent-operator/pull/4#discussion_r142758317. Is there a known stability issue with ops specifically that we'd need to pin the exact version?

Reply:
I'm afraid that link isn't working for me

Reply:
I don't have a strong opinion on either approach, I just think that we should be consistent on whether we pin the exact version or just the major version. Let's wait for David and others to give more input on this and then we can decide whether to pin the major version or the exact version.

Reply:
Since we're working on a spec to define this for the team let's pick an approach for now and adjust once the spec is done. It might take some time to finalise the spec

Reply:
Sounds good, thanks, I think we can pin the exact version of all of the dependencies for now.

Reply:
I pinned all dependencies to their exact version.